test/unit/mem/copy/src/BufSlicing.cpp

```diff
@@ -161,6 +166,9 @@ TEMPLATE_LIST_TEST_CASE("memBufSlicingTest", "[memBuf]", TestAccWithDataTypes)
         ptrNative[i] = static_cast<Data>(mappedTo1D);
     }
 
+    // Finish device tasks before comparing results.
+    slicingTest.synchronizeQueue();
```

Comment:
This is not required because we use always a blocking queue.

Reply:
Never the less I will keep this PR alive to test if our CI working. We are currently updating the CI node.